config/default/editor.editor.basic_html.yml

```diff
@@ -28,8 +28,9 @@ settings:
         -
           name: Links
           items:
-            - DrupalLink
-            - DrupalUnlink
+            - Link
```

Comment:
Was this change cause by a module update?

Reply:
Swapping the core DrupalLink for the link button provided in https://www.drupal.org/project/anchor_link